ravenframework/SupervisedLearning/SupervisedLearning.py

```diff
@@ -280,6 +284,7 @@ def _localNormalizeData(self,values,names,feat):
       @ Out, None
     """
     self.muAndSigmaFeatures[feat] = mathUtils.normalizationFactors(values[names.index(feat)])
+    self.muAndSigmaTargets[self.target[0]] = mathUtils.normalizationFactors(values[names.index(self.target[0])])
```

Comment:
Two comments here: 
First, the normalization for target is only based on the first target, I guess we want to normalize each target.
Second, I think either we need to rewrite the _localNormalizeData to handle both feature and target correctly, or have separate methods for feature and target.